build-ps/jenkins/build-binary

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o xtrace
+
+
+# ------------------------------------------------------------------------------
+# Declate all input variables
+# ------------------------------------------------------------------------------
+JOB_CMAKE=${JOB_CMAKE:-cmake}
+COMPILER=${COMPILER:-default}
+CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:-RelWithDebInfo}
+WITH_TOKUDB=${WITH_TOKUDB:-ON}
+WITH_ROCKSDB=${WITH_ROCKSDB:-ON}
+WITH_EMBEDDED_SERVER=${WITH_EMBEDDED_SERVER:-ON}
+WITH_RAPID=${WITH_RAPID:-ON}
+WITH_SSL=${WITH_SSL:-system}
+WITHOUT_PERFSCHEMA_STORAGE_ENGINE=${WITHOUT_PERFSCHEMA_STORAGE_ENGINE:-OFF}
+ANALYZER_OPT=${ANALYZER_OPT:-}
+MAKE_OPTS=${MAKE_OPTS:--j$(grep -c ^processor /proc/cpuinfo) VERBOSE=1}
```

Comment:
Do not need VERBOSE=1 for every build - too much output.
I would make another checkbox from this param with the value OFF by default.